cmd/commands/cli/command_identities.go

```diff
@@ -306,6 +313,91 @@ func (c *cliApp) getReferralCode(actionArgs []string) error {
 	return nil
 }
 
+const usageSetBeneficiary = "beneficiary <identity> <new beneficiary>"
+
+func (c *cliApp) setBeneficiary(actionArgs []string) error {
```

Comment:
We no longer settle, this should probably just call `set payout address` in tequila and be done with it